frontend/src/app/search/SearchForm.tsx

```diff
@@ -64,25 +72,32 @@ export function SearchForm({
                 }
                 initialQueryParams={sortbyQueryParams}
               />
-              <SearchPagination
-                initialQueryParams={pageQueryParams}
-                formRef={formRef}
-                showHiddenInput={true}
-                totalPages={searchResults.pagination_info.total_pages}
-              />
+              {searchResults.data.length >= 1 ? (
+                <SearchPagination
+                  totalPages={searchResults.pagination_info.total_pages}
+                  page={page}
+                  handlePageChange={handlePageChange}
+                  showHiddenInput={true}
+                  paginationRef={topPaginationRef}
+                />
+              ) : null}
+
               <SearchResultsList
                 searchResults={searchResults.data}
                 maxPaginationError={maxPaginationError}
               />
-              <SearchPagination
-                initialQueryParams={pageQueryParams}
-                formRef={formRef}
-                totalPages={searchResults.pagination_info.total_pages}
-              />
+              {searchResults.data.length >= 1 ? (
+                <SearchPagination
+                  totalPages={searchResults.pagination_info.total_pages}
+                  page={page}
+                  handlePageChange={handlePageChange}
+                />
+              ) : null}
             </div>
           </div>
         </div>
       </div>
+      <input type="hidden" name="fieldChanged" ref={fieldChangedRef} />
```

Comment:
tells the backend/server components which field has actually changed - right now just pagination works with it